src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -9,20 +9,217 @@
+        public static Vector64<byte> Create(byte e0, byte e1, byte e2, byte e3, byte e4, byte e5, byte e6, byte e7) { throw null; }
+        public static Vector64<short> Create(short e0, short e1, short e2, short e3) { throw null; }
+        public static Vector64<int> Create(int e0, int e1) { throw null; }
+        public static Vector64<sbyte> Create(sbyte e0, sbyte e1, sbyte e2, sbyte e3, sbyte e4, sbyte e5, sbyte e6, sbyte e7) { throw null; }
+        public static Vector64<float> Create(float e0, float e1) { throw null; }
+        public static Vector64<ushort> Create(ushort e0, ushort e1, ushort e2, ushort e3) { throw null; }
+        public static Vector64<uint> Create(uint e0, uint e1) { throw null; }
+        public static Vector64<byte> CreateScalar(byte value) { throw null; }
+        public static Vector64<short> CreateScalar(short value) { throw null; }
+        public static Vector64<int> CreateScalar(int value) { throw null; }
+        public static Vector64<sbyte> CreateScalar(sbyte value) { throw null; }
+        public static Vector64<float> CreateScalar(float value) { throw null; }
+        public static Vector64<ushort> CreateScalar(ushort value) { throw null; }
+        public static Vector64<uint> CreateScalar(uint value) { throw null; }
+        public static Vector64<byte> CreateScalarUnsafe(byte value) { throw null; }
+        public static Vector64<short> CreateScalarUnsafe(short value) { throw null; }
+        public static Vector64<int> CreateScalarUnsafe(int value) { throw null; }
+        public static Vector64<sbyte> CreateScalarUnsafe(sbyte value) { throw null; }
+        public static Vector64<float> CreateScalarUnsafe(float value) { throw null; }
+        public static Vector64<ushort> CreateScalarUnsafe(ushort value) { throw null; }
+        public static Vector64<uint> CreateScalarUnsafe(uint value) { throw null; }
+    }
     [StructLayout(LayoutKind.Sequential, Size = 8)]
     public readonly struct Vector64<T> where T : struct
     {
-         private readonly int _dummy;
+        private readonly int _dummy;
+        public static Vector64<T> Zero { get { throw null; } }
+        public Vector64<U> As<U>() where U : struct { throw null; }
+        public Vector64<byte> AsByte() { throw null; }
+        public Vector64<double> AsDouble() { throw null; }
+        public Vector64<short> AsInt16() { throw null; }
+        public Vector64<int> AsInt32() { throw null; }
+        public Vector64<long> AsInt64() { throw null; }
+        public Vector64<sbyte> AsSByte() { throw null; }
+        public Vector64<float> AsSingle() { throw null; }
+        public Vector64<ushort> AsUInt16() { throw null; }
+        public Vector64<uint> AsUInt32() { throw null; }
+        public Vector64<ulong> AsUInt64() { throw null; }
+        public T GetElement(int index) { throw null; }
+        public Vector64<T> SetElement(int index, T value) { throw null; }
```

Comment:
This doesn't mutate the underlying vector; it returns a new vector with the element at the specified index changed. If this is a builder pattern it should probably use `With` instead of `Set`.